entity-framework/core/miscellaneous/cli/dotnet.md

```diff
@@ -124,7 +124,7 @@ Entity Framework Core .NET Command-line Tools 2.1.3-rtm-32065
 
 ## Updating the tools
 
-Use `dotnet tool update --global dotnet-ef` to update the tools to the latest available version, install a specific version by appending `--version <VERSION>` to your command.
+Use `dotnet tool update --global dotnet-ef` to update the global tools to the latest available version, if you have the tools installed locally in your project use `dotnet tool update dotnet-ef`. Install a specific version by appending `--version <VERSION>` to your command.
```

Comment:
We could also add a link to https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-tool-update

Reply:
@smitpatel is [4c90bb7](https://github.com/dotnet/EntityFramework.Docs/pull/2395/commits/4c90bb7ec3258ff4c9164249d86e99065570d022) ok?